worker-workflow/src/main/java/com/github/cafdataprocessing/WorkflowEvaluator.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing;
+
+import com.hpe.caf.worker.document.DocumentWorkerConstants;
+import com.hpe.caf.worker.document.JavaScriptDocumentPostProcessor;
+import com.hpe.caf.worker.document.exceptions.PostProcessingFailedException;
+import com.hpe.caf.worker.document.model.Document;
+import com.hpe.caf.worker.document.model.ResponseOptions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Evaluates a JavaScript workflow against a document, updating document based on matched actions in the workflow
+ */
+public class WorkflowEvaluator
```

Comment:
Add a `private` empty default constructor to avoid this class being instantiated.

Reply:
OK